docs/guide/sass-modern-compiler.md

```diff
@@ -7,3 +7,63 @@ From version `0.17.0`, this module will configure Nuxt to use the new SASS moder
 If the `sass-embedded` dependency is not installed, the module will configure the `modern` compiler for you. In case you get errors, enable the `disableModernSassCompiler` option in the module configuration to fall back to the `legacy` compiler.
 
 Check [Build Performance](https://vuetifyjs.com/en/features/sass-variables/#build-performance) in Vuetify docs for more details.
+
+## Overriding SASS Variables
+
+Vuetify allows for [overriding global and component-level SASS variables](https://vuetifyjs.com/en/features/sass-variables/).  Setting these up requires configuration that is slightly different from the Vuetify
+documentation while using this Nuxt module.
+
+1)  In your styles directory (for this example, we'll use `${workspace}/assets/styles`), create two files - `assets/styles/main.scss` and `assets/styles/settings.scss`
+
+2)  In the `main.scss` file, we'll want to add
+```scss
+@use 'vuetify' with (
+    // Global Vuetify SASS variable overrides.  For example:
+    // $utilities: false,
+    // $reset: false,
+    // $color-pack: false,
+    // $body-font-family: sans-serif
+)
+```
+
+3)  In the `settings.scss` file, we'll want to add
+```scss
+// $button-text-transform-override: capitalize;
+
+@forward 'vuetify/settings' with (
+    // Component Vuetify SASS variable overrides.
+    // For example -- overriding button font capitalization (as seen at the bottom of the v-btn guide here https://vuetifyjs.com/en/api/v-btn/):
+    // $button-text-transform: $button-text-transform-override,
+);
+```
+
+4)  In your `nuxt.config.ts`, add a `css` entry to the `defineNuxtConfig` configuration object that points to `main.scss` like so:
+```javascript
+export default defineNuxtConfig({
+  css: ['assets/styles/main.scss']
+  // other options
+})
+```
+
+> [!TIP]
+> This will import `main.scss` into ALL of your components by default. If you would like different behavior, you can create a different file with these overrides and import that instead.
```

Comment:
Pardon my ignorance, I don't under this tip. Is it referring to the fact that the Global Variables override will "apply" (instead of "import") globally to all components? It might help to add an example of the different behaviour.